resources/testdata/README.md

```diff
@@ -0,0 +1,5 @@
+* `resources/testdata/glyphs3/Oswald-AE-comb.glyphs`
+   * Oswald with most of the glyphs deleted
+   * After deletion go to Info > Features
+      * Update language systems
+      * Delete features
```

Comment:
this is all already done I imagine? Are these instructions intended as general instructions for how to make a test font like this?

Reply:
For that or if you want to build a new version of this file with more glyphs

Reply:
In general I've never regretted capturing how a given test file was produced and have definitely regretted *not* doing so :)

Reply:
oh definitely, I was just confused by the wording. :)